app/views/layouts/_footer.html.erb

```diff
@@ -19,7 +19,7 @@
           <li><a href="/mentorship">Software Mentorship</a></li>
           <li><a href="/scholarships">Scholarship Program</a></li>
           <li><a href="/employers">Employer Services</a></li>
-          <li><a href="/deploy">Web Application Development</a></li>
+          <li><a href="http://deployopcode.io/">Web Application Development</a></li>
```

Comment:
Ah - that's a good point. Probably better for SEO to link to it directly. Let's keep it like you have it.